yarn-project/end-to-end/src/e2e_non_contract_account.test.ts

```diff
@@ -1,54 +0,0 @@
-import { ExtendedNote, Fr, type Logger, Note, type Wallet } from '@aztec/aztec.js';
```

Comment:
This test was just weird - it contains just one test case which looks useless at this point and the test case has nothing to do with non_contract_accounts. So I just nuked it all.